src/db/sql/events-actions/queries.ts

```diff
@@ -28,7 +28,6 @@ function fullChainCTE(db_client: postgres.Sql, from?: string, to?: string) {
     FROM
       blocks b
       INNER JOIN pending_chain ON b.id = pending_chain.parent_id
-      AND pending_chain.id <> pending_chain.parent_id
```

Comment:
I am not confident that this can never be true.  I just don't know the Mina spec well enough.  What is gained by removing this filter?  Is the performance notably better?